OSDOCS#6018: Authenticating cert-manager Operator on the AWS STS cluster (#59717)
openshift/openshift-docs: security/cert_manager_operator/cert-manager-authenticate-aws.adoc

:_content-type: ASSEMBLY
[id="cert-manager-authenticate-aws"]
= Authenticating the {cert-manager-operator} with AWS Security Token Service
include::_attributes/common-attributes.adoc[]
:context: cert-manager-authenticate-aws

toc::[]

> You can authenticate the {cert-manager-operator} on the AWS Security Token Service (STS) cluster. You can configure cloud credentials for the {cert-manager-operator} by using the `ccoctl` binary

Reviewer: ```suggestion
You can authenticate the {cert-manager-operator} on the AWS Security Token Service (STS) cluster. You can configure cloud credentials for the {cert-manager-operator} by using the `ccoctl` binary.
```

Reviewer: LGTM otherwise, thanks!

Author: Addressed, thank you

> You can authenticate the {cert-manager-operator} with AWS Security Token Service (STS). You can configure cloud credentials for the {cert-manager-operator} on the AWS Security Token Service cluster.

Reviewer: ```suggestion
You can authenticate the {cert-manager-operator} on an AWS Security Token Service (STS) cluster. The cloud credentials on the cluster can be configured with the help of the `ccoctl` binary.
```

:_content-type: ASSEMBLY
[id="cert-manager-authenticate-aws"]
= Authenticating the {cert-manager-operator} with AWS Security Token Service
include::_attributes/common-attributes.adoc[]
:context: cert-manager-authenticate-aws

toc::[]

> You can authenticate the {cert-manager-operator} on the AWS Security Token Service (STS) cluster. You can configure cloud credentials for the {cert-manager-operator} by using the `ccoctl` binary.

Reviewer 2: ```suggestion
You can authenticate the {cert-manager-operator} on the AWS Security Token Service (STS) cluster with the cloud credentials. You can configure cloud credentials for the {cert-manager-operator} by using the `ccoctl` binary.
```
